sql/core/src/test/resources/sql-tests/results/explain-aqe.sql.out

```diff
@@ -1086,20 +1086,20 @@ struct<plan:string>
    +- 'UnresolvedRelation [explain_temp4], [], false
 
 == Analyzed Logical Plan ==
-InsertIntoHadoopFsRelationCommand Location [not included in comparison]/{warehouse_dir}/explain_temp5], Append, `spark_catalog`.`default`.`explain_temp5`, org.apache.spark.sql.execution.datasources.CatalogFileIndex@7d811218, [key, val]
+InsertIntoHadoopFsRelationCommand Location [not included in comparison]/{warehouse_dir}/explain_temp5], Append, `spark_catalog`.`default`.`explain_temp5`, org.apache.spark.sql.execution.datasources.CatalogFileIndex@x, [key, val]
```

Comment:
shall we simply override `toString` in `FileIndex`? One idea is to produce `className(rootPaths.mString(","))`

Reply:
```
-InsertIntoHadoopFsRelationCommand Location [not included in comparison]/{warehouse_dir}/explain_temp5], Append, `spark_catalog`.`default`.`explain_temp5`, org.apache.spark.sql.execution.datasources.CatalogFileIndex@7d811218, [key, val]
+InsertIntoHadoopFsRelationCommand Location [not included in comparison]/{warehouse_dir}/explain_temp5), [key, val]
```

After override `toString` in `FileIndex`, `Append, `spark_catalog`.`default`.`explain_temp5`, org.apache.spark.sql.execution.datasources.CatalogFileIndex@7d811218` disappeared from result

Reply:
So, it means we cannot do that. Did I understand correctly? Given that, I'm +1 with the AS-IS PR.

Reply:
@cloud-fan `className(rootPaths.mString(","))` will print out the path related to the running environment, like 
```
org.apache.spark.sql.execution.datasources.CatalogFileIndex(file:/Users/userNaame/spark-source/sql/core/spark-warehouse/org.apache.spark.sql.SQLQueryTestSuite/explain_temp5)
```
It doesn't look good

Reply:
> So, it means we cannot do that. Did I understand correctly? Given that, I'm +1 with the AS-IS PR.

Only override the `toString` method cannot achieve the goal, and as I said above, `rootPaths.mString(",")` needs further cleaning to avoid differences in test execution paths, so I prefer the current way.